src/mudlet.cpp

```diff
@@ -1098,6 +1103,7 @@ void mudlet::slot_close_profile_requested(int tab)
     }
 }
 
+// Not currently used - may not be properly functional anymore!
 void mudlet::slot_close_profile()
```

Comment:
Dead code to delete?

Reply:
No, I do want to bring it into play, see above.